src/main.tsx

```diff
@@ -5,16 +5,20 @@ import App from './App';
 import './index.css';
 import GlobalStateProvider from './state/contexts/GlobalState';
 import RegistrationStateProvider from './state/contexts/RegistrationState.js';
+import TransactionStateProvider from './state/contexts/TransactionState';
 import { reducer, registrationReducer } from './state/reducers';
+import { transactionReducer } from './state/reducers/TransactionReducer';
 // setup sentry
 import './utils/sentry';
 
 ReactDOM.createRoot(document.getElementById('root') as HTMLElement).render(
   <React.StrictMode>
     <GlobalStateProvider reducer={reducer}>
-      <RegistrationStateProvider reducer={registrationReducer}>
-        <App />
-      </RegistrationStateProvider>
+      <TransactionStateProvider reducer={transactionReducer}>
+        <RegistrationStateProvider reducer={registrationReducer}>
+          <App />
+        </RegistrationStateProvider>
+      </TransactionStateProvider>
```

Comment:
see comment related to approaches we can take - imo, i feel `TransactionStateProvider` should be available globally in the app and not just around the `Transaction` page.

Reply:
agreed, moved it there